sci-physics/vgm/vgm-4.4.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Similarly to your other PR, this looks like a case for revbump.

Reply:
Ok, done! 
I have added the patch which fixes (re)build for those on 17.1 also to the non-revbumped ebuild, so the revbump is for the EAPI bump alone.

Reply:
Just leave the old ebuild alone, you are doing a version bump anyway.

Reply:
Ok, done!